### DISCUSSION TITLE
PRO achievement not showing up

### DISCUSSION THREAD

**Discussion by onwp** [Schweinepriester/github-profile-achievements#645] [Q&A] [answered]
Hi,

I bought a team subscription on https://github.com/pricing 

But no PRO achievement showed up on my profile. 

Does anyone know the reason or did it happen to anyone else too?

**Comment by runepiper** [comment, marked as answer]
There is no **PRO** achievement. As of today it already appears under your **Highlights** section on your profile:

![Screenshot 2023-02-22 at 19 13 31](https://user-images.githubusercontent.com/4521147/220718920-cf78c40c-0772-4784-b8f3-65a6c862a841.png)

**Reply by onwp** [reply]
Mispronounced. I meant Highlights. Thanks, I can see it too now.
